terraform/environments/prd/aws/files/cloud-configs/bank.yaml

```diff
@@ -0,0 +1,14 @@
+#cloud-config
+package_update: true
+package_upgrade: true
+packages: 
+  - python3-pip
+  - git
+write_files:
+  - content: |
+      ggramal
+    path: /tmp/vp.ini
+runcmd:
+  - pip install ansible
+  - cd /tmp/ && git clone --branch features-hw-26 https://github.com/ESergiusz/dos14-Eremeev-GitFlow.git authz
```

Comment:
lest make branch settable

Reply:
+ bank repo is https://github.com/OpsStrike no?

Reply:
@ggramal done